cmd/tracee-perf/main.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	prometheusAddressFlag = "prometheus"
+	periodFlag            = "period"
+	outputFlag            = "output"
+)
+
+type measurment struct {
+	AvgEbpfRate       float64 `json:"avgEbpfRate"`
+	AvgLostEventsRate float64 `json:"avgLostEventsRate"`
+	LostEvents        int     `json:"lostEvents"`
+}
+
+func (m measurment) Print() {
+	log.Printf("\n")
+	fmt.Printf("Events/Sec:     %f\n", m.AvgEbpfRate)
+	fmt.Printf("EventsLost/Sec: %f\n", m.AvgLostEventsRate)
+	fmt.Printf("Events Lost:    %d\n", m.LostEvents)
+	fmt.Println("===============================================")
+}
+func (m measurment) PrintJson() {
+	res, _ := json.Marshal(m)
+	fmt.Println(string(res))
+}
+
+type OutputMode string
+
+const (
+	jsonOutput   OutputMode = "json"
+	prettyOutput OutputMode = "pretty"
+)
+
+func main() {
+	app := cli.App{
+		Name:  "tracee-perf",
+		Usage: "A prometheus based performance probe for tracee",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  prometheusAddressFlag,
+				Usage: "address of a prometheus instance tracking tracee",
+				Value: "http://localhost:9090",
+			},
+			&cli.IntFlag{
+				Name:  periodFlag,
+				Usage: "period of scraping in seconds",
+				Value: 5,
+			},
+			&cli.StringFlag{
+				Name:  outputFlag,
+				Usage: "set output format (options: pretty, json)",
+				Value: "pretty",
+			},
+		},
+		Action: func(ctx *cli.Context) error {
+			address := ctx.String(prometheusAddressFlag)
+
+			if address == "" {
+				return fmt.Errorf("prometheus address required for tracee-er")
```

Comment:
CHORE: What is tracee-er ? We should normalize "tracee-perf" or something else I think. I like "tracee-perf", but it gives us the sensation we're doing profiling (instead of tracing). Tracee-perfprobe or something else ? WDYT ?